pallets/gear/src/lib.rs

```diff
@@ -1109,82 +1087,39 @@ pub mod pallet {
                                     log::debug!(
                                         "Can not instrument code '{:?}' for program '{:?}'",
                                         code_id,
-                                        program_id
+                                        dispatch.destination()
                                     );
                                     continue;
                                 }
                             } else {
```

Comment:
What about adding this comment?
```suggestion
                            } else {
                            // This branch is considered unreachable, because there can't be a program without code.
                            // Reaching this branch is a sign of a serious storage/extrinsics corruption. 
```

Reply:
In general, this PR does not affect any code storing logic, its just fast-find beautification, so I prefer to delay any explanations for future PRs, because everything can change.

Reply:
This invariant didn't change for a long time. And seems not to be in the future. Anyway, in 'process_queue' function we have different unclear logical branches. I mean it's not obvious whether they are reachable and in what situations. So better to write this and change, rather than forget about it.